### PR TITLE
AP-3388: Add the default Level of Service and Scopes page

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3388)

This enables the first part of the extended loop -
the page that displays the Default level of service and scope for the current proceeding, client role and delegated function combo
It requests those defaults from the new LFA endpoint and records the values if the user selects yes
If the user selects no, it records that choice and leaves the values blank to be completed by subsequent tickets

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/providers/proceeding_loop/emergency_defaults/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      local: true
+    ) do |form| %>
+  <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
+  <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+  <%= page_template template: :basic, form: form do %>
+    <%= form.govuk_radio_buttons_fieldset :accepted_emergency_defaults,
+                                          inline: true,
+                                          legend: { size: 'l', tag: 'h2', text: t('.question') } do %>
+      <p>
+        <strong>Default level of service:</strong> <%= form.object.emergency_level_of_service_name %>
```
Comment (kmahern) [inline app/views/providers/proceeding_loop/emergency_defaults/show.html.erb:14]: Should these be translations?

File: app/views/providers/proceeding_loop/substantive_defaults/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      local: true
+    ) do |form| %>
+  <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
+  <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+  <%= page_template template: :basic, form: form do %>
+    <%= form.govuk_radio_buttons_fieldset :accepted_substantive_defaults,
+                                          inline: true,
+                                          legend: { size: 'l', tag: 'h2', text: t('.question') } do %>
+      <p>
+        <strong>Default level of service:</strong> <%= form.object.substantive_level_of_service_name %>
```
Comment (kmahern) [inline app/views/providers/proceeding_loop/substantive_defaults/show.html.erb:14]: Should these be translations?

Comment (colinbruce) [reply app/views/providers/proceeding_loop/substantive_defaults/show.html.erb:14]: Yes, they should be.  Good catch.  Feel free to update if this blocks your subsequent tickets :+1:

File: app/services/flow/proceeding_loop.rb [hunk 3]
```diff
@@ -59,7 +72,7 @@ def date_confirmation_required
     alias_method :date_confirmation_required?, :date_confirmation_required
 
     def at_end_of_loop?
-      final_proceeding_in_loop.id == @application.provider_step_params["id"] && %w[delegated_functions confirm_delegated_functions_date].include?(@application.provider_step)
+      final_proceeding_in_loop.id == @application.provider_step_params["id"] && controllers[-2..].include?(@application.provider_step)
```
Comment (kmahern) [inline app/services/flow/proceeding_loop.rb:75]: I found this a little opaque, possibly worth a comment/explaining that the order of the controller loops matters?

**Review by kmahern — APPROVED** [review]
A few minor comments/suggestions but no blockers

**Review by kmahern — APPROVED** [review]
have reviewed the new comits and lgtm 👍

**Pull request merged**